### ISSUE TITLE
Make Login Page

### ISSUE THREAD

**Issue by batflarrow** [iiitv/CTF-frontend#4]

**Comment by B1TSH3LL** [comment]
Hey I Can Make A Pure HTML CSS Login Page For You Including JS For Credentials Without any Database Would You Like To Assig It To Me???

**Comment by batflarrow** [comment]
> Hey I Can Make A Pure HTML CSS Login Page For You Including JS For Credentials Without any Database Would You Like To Assign It To Me???

We already have a backend for this portal so can you try and make the login page as a react component following the designs provided..?

**Comment by Bianka-ui** [comment]
Hi! Can I work on this? According to your previous comment I can do it. @batflarrow

**Comment by Safwan310** [comment]
Hey @batflarrow , I looked at the figma design and it seem like I can do this. Please assign it to me. Thank you

**Comment by batflarrow** [comment]
> Hi! Can I work on this? According to your previous comment I can do it. @batflarrow
Yeah go ahead Just make sure to create Components and Containers folder separate inside the src directory and work inside them @Bianka-ui

**Comment by batflarrow** [comment]
> Hey @batflarrow , I looked at the figma design and it seem like I can do this. Please assign it to me. Thank you

Hey I have added another issue can you go ahead with that..? @Safwan310 Issue #6

**Comment by Safwan310** [comment]
> > Hey @batflarrow , I looked at the figma design and it seem like I can do this. Please assign it to me. Thank you
> 
> Hey I have added another issue can you go ahead with that..? @Safwan310 Issue #6

Sure

**Comment by Bianka-ui** [comment]
> > Hi! Can I work on this? According to your previous comment I can do it. @batflarrow
> > Yeah go ahead Just make sure to create Components and Containers folder separate inside the src directory and work inside them @Bianka-ui

Sure you can assign me. @batflarrow

**Comment by batflarrow** [comment]
> > > Hi! Can I work on this? According to your previous comment I can do it. @batflarrow
> > > Yeah go ahead Just make sure to create Components and Containers folder separate inside the src directory and work inside them @Bianka-ui
> 
> Sure you can assign me. @batflarrow

Sure go ahead you have been assigned the issue!

**Comment by batflarrow** [comment]
Any updates @Bianka-ui ...?

**Comment by siddhantsingh186** [comment]
Can I work on this @batflarrow ?

**Comment by batflarrow** [comment]
> Can I work on this @batflarrow ?

Sure go ahead @siddhantsingh186
